sdk/keyvault/azkeys/client.go

```diff
@@ -23,7 +23,7 @@ import (
 // Client is the struct for interacting with a KeyVault Keys instance
 type Client struct {
 	kvClient *generated.KeyVaultClient
-	vaultUrl string
+	vaultURL string
```

Comment:
Weren't you also going to expose the `VaultURL` for debugging? This would likely be better as a `func` to avoid mutability, which would be disastrous once the credential token is initially fetched.